app/scripts/lib/marketing-email-client.js

```diff
@@ -30,7 +30,8 @@ define([
         url: url,
         type: method,
         accessToken: accessToken,
-        data: data
+        data: data,
+        timeout: 2500
```

Comment:
Let's make 2500 a named constant like `DEFAULT_XHR_TIMEOUT_MS` and allow an override to be passed in to the `MarketingEmailClient` constructor.